data/paver/skeleton/templates/index.html.template

```diff
@@ -1,9 +1,9 @@
-<!doctype html>
+{% raw %}<!doctype html>
 <html>
-    <head><title>$(( plugin ))$ skeleton page</title></head>
+    <head><title>{{ plugin }} skeleton page</title></head>
```

Comment:
One change we made awhile back was to swap out our overridden Jinja2 default tags with the actual defaults that Jinja2 ships.  However, at the time we neglected to update our paver skeleton template - here's that change.

I'm not sure that anyone actually uses the ``paver create_plugin`` task, but I do try to keep it up to date since it's what the Sideboard tutorial has people do.